### ISSUE TITLE
[BUG] WSLManager opens in a hang state

### ISSUE THREAD

**Issue by lynxul25** [bostrot/wsl2-distro-manager#262]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When opening the WSLM, version 1.8.14 the window remains blank and unresponsive. Only End task from Taskmanager ends the thread.

All was working well until this behaviour started.

![Screenshot 2024-10-10 131755](https://github.com/user-attachments/assets/30f3bd8a-c6a3-4e46-99d7-e6df571dad2d)

### Steps To Reproduce

Start the WSLManager

### Environment

```markdown
- Windows Version: 11
- WSL Distro Manager Version: 1.8.14
- WSL Version: 2
- Source: Github
```

**Comment by mmcer** [comment]
i have the same issue today,
environment: win11, WSL2

i remember yesterday i can use this app, but i installed OS updates, maybe this involved new conflict with app.

i try to find if there is log file created, but not found.

CPU usage continue keeping ~27%

**Comment by mmcer** [comment]
the temp solution is: remove shared_preferences.json file from C:\Users\[current user]\AppData\Roaming\com.bostrot\WSL Distro Manager
then restart the app.

**Comment by mir-am** [comment]
I also have the same problem. The app starts in a hang state and the window is just white.
OS: Windows 11
WSL: 2

**Comment by aimzieslol** [comment]
Having the same problem, also. Installed via both `scoop` and `choco`, uninstalled/re-installed multiple times, run as `admin`, etc. etc. No luck. Worked great up until a few days ago.

OS = Windows 11
WSL = 2

**Comment by silasalves** [comment]
Same here, I installed with choco and it doesn't open. It shows a white window and then closes.

**Comment by bostrot** [comment]
> the temp solution is: remove shared_preferences.json file from C:\Users[current user]\AppData\Roaming\com.bostrot\WSL Distro Manager then restart the app.

@aimzieslol @mir-am @silasalves have you tried the solution above?
Also please try the latest version from the releases page.

**Comment by aimzieslol** [comment]
> > the temp solution is: remove shared_preferences.json file from C:\Users[current user]\AppData\Roaming\com.bostrot\WSL Distro Manager then restart the app.
> 
> @aimzieslol @mir-am @silasalves have you tried the solution above? Also please try the latest version from the releases page.

@bostrot Downloading the app fresh (I used the portable/zip version) works fine. I tried deleting `shared_preferences.json` on another computer and it crashes right away. I can provide you the `.dmp` file if that would help?

**Comment by mmcer** [comment]
> > > the temp solution is: remove shared_preferences.json file from C:\Users[current user]\AppData\Roaming\com.bostrot\WSL Distro Manager then restart the app.
> > 
> > 
> > @aimzieslol @mir-am @silasalves have you tried the solution above? Also please try the latest version from the releases page.
> 
> @bostrot Downloading the app fresh (I used the portable/zip version) works fine. I tried deleting `shared_preferences.json` on another computer and it crashes right away. I can provide you the `.dmp` file if that would help?

After i use the temp solution, it works, and recently app pop up a new version(latest), after download, the latest version reproduce the blank page issue, and this time, delete "shared_preferences.json" file doesn't work any more, so i have to rollback to the previous version for Windows.

By the way, i use portable version too.

**Comment by mehalick** [comment]
Same here. I installed today from Microsoft Store and used once before rebooting. After reboot, it hangs but uses 10 - 20% CPU.

I don't have the directory C:\Users[current user]\AppData\Roaming\com.bostrot\WSL Distro Manager, so cannot delete shared_preferences.json.

I uninstalled and reinstalled from Microsoft Store and it seems to be working again.

**Comment by bostrot** [comment]
Probably has something to do with this https://github.com/flutter/flutter/issues/89211

I implemented some corruption fixing or in worst case re-creation of the shared_preferences.json file. Will put this in the next release.

**Comment by HK-hub** [comment]
I agree that it crashes a few seconds after opening and no directory C:\Users[current user]\AppData\Roaming\com.bostrot\WSL Distro is produced

![Image](https://github.com/user-attachments/assets/8ed4450c-fedf-4611-80c1-81c8f8c39620)

![Image](https://github.com/user-attachments/assets/1a5ce596-a090-46f2-990a-5fb6319fea60)
